src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -35,6 +40,64 @@ public IndexModel createIndex(CreateIndexRequest createIndexRequest) throws Pine
         return indexModel;
     }
 
+    public <E extends Enum<E>> boolean validateEnums(String stringToValidate, List<E> validEnums) {
```

Comment:
This method doesn't need to be public, I don't think? Also, I'd suggest moving it down towards the bottom of the class.

Reply:
I actually had it w/o any access modifier previously, but my tests wouldn't run because my IDE said it had to be public.

Is there something special I need to do? I'm using IntelliJ and I've marked the dir as a 'test source root'...